docs/assets/js/charts.js

```diff
@@ -1,703 +1,835 @@
+        width = +svg.attr('width')-2*pad,
+        height = +svg.attr('height')-2*pad,
+        outerRadius = Math.min(width, height) * 0.5 - 200,
+        innerRadius = outerRadius - 50;
+
+    //Initialize chord diagram
+    const chord = d3.chord()
+        .padAngle(0.05)
+        .sortSubgroups(d3.descending);
+
+    const g = svg.append('g')
+        .attr('transform', 'translate(' + (width / 2 + pad) + ',' + (height / 2 + pad) + ')')
+        .datum(chord(matrix));
+
+    // Draw the ribbons that go from group to group
+    const ribbon = d3.ribbon()
+        .radius(innerRadius);
+
+    const ribbons = g.append('g')
+        .attr('class', 'ribbons')
+        .selectAll('path')
+        .data(function(chords)
+        {
+            return chords;
+        })
+        .enter().append('g')
+        .on('mouseover', fadeRibbon(.1))
+        .on('mouseout', fadeRibbon(1));
+
+    ribbons
+        .append('path')
+        .attr('d', ribbon)
+        .style('stroke-width', 5.0)
+        .style('stroke', function()
+        {
+            return '#ffffff';
+        });
+
+    ribbons
+        .append('path')
+        .attr('d', ribbon)
+        .style('fill', function(d)
+        {
+            return extendedChartColor(d.source.index);
+        })
+        .style('stroke-width', 2.0)
+        .style('stroke', function(d)
+        {
+            return extendedChartColor(d.source.index);
+        });
+
+    ribbons.append('title')
+        .text(function(d)
+        {
+            return ribbonTip(d);
+        });
+
+    // Defines each "group" in the chord diagram
+    const group = g.append('g')
+        .attr('class', 'groups')
+        .selectAll('g')
+        .data(function(chords)
+        {
+            return chords.groups;
+        })
+        .enter().append('g');
+
+    // Draw the radial arcs for each group
+    const arc = d3.arc()
+        .innerRadius(innerRadius)
+        .outerRadius(outerRadius);
+
+    group.append('path')
+        .style('fill', function(d)
+        {
+            return extendedChartColor(d.index);
+        })
+        .style('stroke-width', 2.0)
+        .style('stroke', function(d)
+        {
+            return extendedChartColor(d.index);
+        })
+        .attr('d', arc)
+        .on('mouseover', fadeRibbonsWithSameSource(.1))
+        .on('mouseout', fadeRibbonsWithSameSource(1))
+        .append('title')
+        .text(function(d)
+        {
+            return chordTip(d);
+        });
+
+    // Add labels to each group
+    group.append('text')
+        .attr('dy', '.35em') //width
+        .attr('class', 'org-label')
+        .attr('transform', function(d, i)
+        {
+            d.angle = (d.startAngle + d.endAngle) / 2;
+            d.name = orgs[i];
+            const degree = d.angle * 180 / Math.PI;
+            const flip = (degree > 180 ? 90 : -90);
+            return 'rotate(' + degree + ')' +
+				'translate(0,' + -1 * (outerRadius + 5) + ')' +
+				'rotate(' + flip + ')';
```

Comment:
Some more tabs 🙂.

Reply:
Weird that this somehow got through, I expected the 'no mixed tabs/spaces' rule to catch this. I explicitly added the `no-tabs` rule now so that should not get in anymore.